homesensorhub/communication/payload.py

```diff
@@ -0,0 +1,55 @@
+"""Module which implements the payload class."""
```

Comment:
Payload and packet concepts should be unified. They should represent the internal data structure of the code.
Sensor modules should output a packet, which is received by the sender modules. These should be no distinction in between.

Reply:
Good point :). I will implement this now.

Reply:
Fixed in 927c7270af33cebf6ac65ba1f8ee05ced149fea6.